cost/google/instances_stackdriver_utilization/google_instances_stackdriver_utilization.pt

```diff
@@ -368,10 +385,10 @@ policy "pol_utilization" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing Google instance StackDriver Utilization data"
     detail_template <<-EOS
 # Google Instance StackDriver Utilization
-| Region | instanceID | Hostname | Current Machine Type | Machine Type To Downsize | CPU Maximum % | CPU Minimum % | CPU Average % | Disk Maximum % | Disk Minimum % | Disk Average % | Memory Minimum % | Memory Maximum % | Memory Average % | Tags |
-| ------ | ---------- | -------- | -------------------- | ------------------------ | ------------- | ------------- | ------------- | -------------- | -------------- | -------------- | ---------------- | ---------------- | ---------------- | ---- |
+| Project ID | Region | instanceID | Hostname | Current Machine Type | Machine Type To Downsize | CPU Maximum % | CPU Minimum % | CPU Average % | Disk Maximum % | Disk Minimum % | Disk Average % | Memory Minimum % | Memory Maximum % | Memory Average % | Tags |
```

Comment:
Sort by projectid then region.

Reply:
I have addressed this comment please review the changes.